mailbox/store/src/test/java/org/apache/james/mailbox/store/GroupFolderResolverTest.java

```diff
@@ -20,55 +20,35 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import java.util.List;
-import java.util.Locale;
-
-import org.apache.james.mailbox.MailboxSession.SessionType;
 import org.apache.james.mailbox.model.MailboxPath;
 import org.apache.james.mailbox.store.mail.model.impl.SimpleMailbox;
+import org.junit.Before;
 import org.junit.Test;
 
 public class GroupFolderResolverTest {
 
     private static final long UID_VALIDITY = 9999;
-    private List<Locale> localePreferences = null;
-    private List<String> sharedSpaces = null;
-    private char pathSeparator = ' ';
-    
+    private GroupFolderResolver testee;
+
+    @Before
+    public void setUp() {
+        testee = new GroupFolderResolver();
+    }
+
     @Test
     public void isGroupFolderShouldReturnFalseWhenMailboxNamespaceIsNull() {
-        SimpleMailboxSession mailboxSession = new SimpleMailboxSession(1, "username", "password", localePreferences, sharedSpaces, null, pathSeparator, SessionType.User);
-        GroupFolderResolver testee =  new GroupFolderResolver(mailboxSession);
-        
         SimpleMailbox mailbox = new SimpleMailbox(new MailboxPath(null, "user", "name"), UID_VALIDITY);
         assertThat(testee.isGroupFolder(mailbox)).isFalse();
     }
     
     @Test
     public void isGroupFolderShouldReturnFalseWhenMailboxNamespaceEqualsToUserNamespace() {
-        SimpleMailboxSession mailboxSession = new SimpleMailboxSession(1, "username", "password", localePreferences, sharedSpaces, null, pathSeparator, SessionType.User);
-        GroupFolderResolver testee =  new GroupFolderResolver(mailboxSession);
-        
         SimpleMailbox mailbox = new SimpleMailbox(MailboxPath.forUser("user", "name"), UID_VALIDITY);
         assertThat(testee.isGroupFolder(mailbox)).isFalse();
     }
     
     @Test
-    public void isGroupFolderShouldReturnFalseWhenMailboxNamespaceEqualsToOtherUsersNamespace() {
```

Comment:
I don't understand why this test is no longer valid

Reply:
how the commit title "Group mailbox resolving can now be done statically" (if i'm correct) relates to that ?

Reply:
Because OtherUserNamespace is a concept that is orthogonal to mailboxPath.

Because a group mailbox is only one that is not part of #private 

Because UserNamespace and shared namespace is a IMAP level concern. THAT is the purpose of the  NAMESPACE related commit. Make it fully a IMAP concern.